### DISCUSSION TITLE
Shelly Plus 1

### DISCUSSION THREAD

**Discussion by awilinsk** [ESPresense/ESPresense#419] [Q&A]
Does anyone know if it's possible to use a Shelly Plus 1 as base station for ESPresense? My thought is that I can stick this in an outlet box and be completely out of sight and powered easily. From what I read, it's based on the ESP32-U4WDH.

**Comment by DTTerastar** [comment]
I have one to try.  It should work.

**Reply by awilinsk** [reply]
Great! If you get it to work, would you please post the flashing process? Mainly what drop down selection you picked on the install page.

**Comment by DTTerastar** [comment]


**Reply by DTTerastar** [reply]
This is wrong it's a single core cpu so it needs it's own firmware flavor.

**Comment by JamesFinlayson-zz** [comment]
Does this stop the plus 1 from working as a smart light switch? I'd love to install ESPresence on mine, but do need to still use them to control the lights.

**Comment by DTTerastar** [comment]
Yes, it would change it.  We'd need to add back in support for turning stuff on and off.

**Reply by mmuenker** [reply]
I see it similar to @JamesFinlayson. That would be a great feature. I am in the process of replacing all light switches with Shelly switches.

Is this feature already included in the roadmap? If not, it would be nice to add this feature to the roadmap.
If it hasn't already been implemented by August, I may start implementing it in September or October.

**Comment by DTTerastar** [comment]
I have the headers to try and a Shell Plus 1.  Just need to figure out how to get wires attached, it's a really small pitch!

**Reply by bachoo786** [reply]
> I have the headers to try and a Shell Plus 1. Just need to figure out how to get wires attached, it's a really small pitch!

Hey did you manage to use shelly 1 plus as a base station after all?

**Comment by TheBlackMini** [comment]
I'm going to do something bad and necro this...

Can the Shelly Plus 1/1PM/2PM be stations?
They are now based on the dual core version of the ESP32-U4WDH

**Reply by DTTerastar** [reply]
ESP32-U4WDH should work okay
